src/diffpy/nmf_mapping/tests/run.py

```diff
@@ -13,7 +13,7 @@
 #
 ##############################################################################
 """Convenience module for executing all unit tests with
-python -m \.tests.run
+python -m diffpy.nmf_mapping.tests.run
```

Comment:
flake8 passes.

Reply:
This is the wrong syntax here as we are moving over to using pytest to run tests.  Please look in cookicutter or one of the other projects

Reply:
From the cookiecutter repo, I see it is the latest `tests/run.py` file:

https://github.com/Billingegroup/cookiecutter/blob/main/%7B%7B%20cookiecutter.repo_name%20%7D%7D/src/%7B%7B%20cookiecutter.package_dir_name%20%7D%7D/tests/run.py

I checked diffpy.structure

https://github.com/diffpy/diffpy.structure/blob/main/src/diffpy/structure/tests/run.py

but it seems even older.

What is the "latest" one?

Reply:
I think we just delete this run.py?   Then in the ci (and when we run it locally) we just run `pytest` or `coverage`